charmander/app.py

```diff
@@ -8,9 +8,9 @@
 from microcosm_secretsmanager.loaders.conventions import load_from_secretsmanager
 
 import charmander.postgres  # noqa
-import charmander.routes.example.controller  # noqa
-import charmander.routes.example.crud   # noqa
-import charmander.stores.example_store  # noqa
+import charmander.routes.pizza.controller  # noqa
```

Comment:
Below I explain that these imports are necessary to make sure the code in the `binding` decorator runs and registers the components in these modules to the `graph.